ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/handlers/HandlerFactory.java

```diff
@@ -47,7 +48,9 @@ public static Handler<RoutingContext> jsonRpcParser() {
   }
 
   public static Handler<RoutingContext> jsonRpcExecutor(
-      final JsonRpcExecutor jsonRpcExecutor, final Tracer tracer) {
-    return JsonRpcExecutorHandler.handler(jsonRpcExecutor, tracer);
+      final JsonRpcExecutor jsonRpcExecutor,
+      final Tracer tracer,
+      final JsonRpcConfiguration configuration) {
```

Comment:
nit: suggestion of variable name to follow the same patter jsonRpcConfiguration

Reply:
done